frontend/src/components/blocks/edit/BlockTimeLabel.tsx

```
@@ -0,0 +1,26 @@
+ * 親ラッパー(.schedule-time-wrapper / .transport-time-wrapper)の flex 方向に応じて
+ * 区切り(横「|」/縦「—」)が CSS で切り替わる。
+ */
+export function BlockTimeLabel({ startTime, endTime }: BlockTimeLabelProps) {
+  if (endTime == null) {
+    return <div className='font-medium text-neutral-700'>{dayjs(startTime).format('HH:mm')}</div>;
+  }
+
+  return (
+    <>
+      <div className='font-medium text-neutral-700'>{dayjs(startTime).format('HH:mm')}</div>
+      <div className='-my-1 horizontal-divider text-center text-gray-400 text-xs leading-none'>|</div>
```

Biome (2.4.16)

[error] These CSS classes should be sorted.

(lint/nursery/useSortedClasses)